src/main/java/core/data/medImage/MedImageAttribute.java

```diff
@@ -0,0 +1,103 @@
+package core.data.medImage;
+
+/**
+ * Stores a single tag that is part of metadata of medical images.
+ */
+public class MedImageAttribute {
+
+    public final String name;
+
+    public final String tag;
+    public final AttributeTagType tagType;
+
+    public final Object value;
+    public final AttributeValueRepresentation valueRepresentation;
+
+    public String description;
+
+    public MedImageAttribute(String name, String tag, AttributeTagType tagType, Object value, AttributeValueRepresentation valueRepresentation) {
+        this.name = name;
+        this.tag = tag;
+        this.tagType = tagType;
+        this.value = value;
+        this.valueRepresentation = valueRepresentation;
+    }
+
+    /**
+     * Creates DICOM data attribute
+     * @param tag attribute tag  - (XXXX,XXXX) with hexadecimal numbers
+     * @param name attribute name
+     * @param valueRepresentation  Value Representation (VR) that describes the data type and format of the attribute value (see DICOM documentation)
+     * @param value attribute value
+     * @return created attribute for Dicom format
+     */
+    public static MedImageAttribute CreateDicomAttribute(String tag, String name, AttributeValueRepresentation valueRepresentation, Object value)
```

Comment:
I think this should be a separate class, smth like DicomAttributeCreator, because this seems to be messing with the idea that MedImage is super class for those. Any class that inherits it would need to change code for MedImageAttribute rather than creating a  custom class, which IMO is not good.
Also, may be possible to create a static class with all static methods to hold just tag creations
Same applies to all creation methods